src/azul/health.py

```diff
@@ -270,10 +278,11 @@ def elasticsearch(self):
 
     @lru_cache
     def _lambda(self, lambda_name) -> JSON:
+        url = config.lambda_endpoint(lambda_name).set(path='/health/basic',
+                                                      args={'catalog': self.catalog})
+        log.info('Requesting %r', url)
+        response = requests.get(str(url))
```

Comment:
```suggestion
        url = str(config.lambda_endpoint(lambda_name).set(path='/health/basic',
                                                      args={'catalog': self.catalog}))
        log.info('Requesting %r', url)
        response = requests.get(url)
```

So we don't log the `furl` instance.